src/crimson/os/seastore/seastore.cc

```diff
@@ -40,10 +40,50 @@ SeaStore::SeaStore(
     transaction_manager(std::move(tm)),
     collection_manager(std::move(cm)),
     onode_manager(std::move(om))
-{}
+{
+  register_metrics();
+}
 
 SeaStore::~SeaStore() = default;
 
+void SeaStore::register_metrics()
+{
+  namespace sm = seastar::metrics;
+  using op_type_t = SeaStore::op_type_t;
+  auto lat_label = sm::label("latency");
+  std::map<op_type_t, sm::label_instance> labels_by_op_type {
```

Comment:
nit,
```c++
std::vector<op_type_t, sm::label_instance> labels_by_op_type[] = {
```

Reply:
build error, it is not array, can't use [], and it is a map not vector.

Reply:
updated. please review again.

Reply:
sorry, i fixed the typo.

Reply:
#42427